packages/itwinui-react/src/react-table/react-table.ts

```diff
@@ -0,0 +1,1323 @@
+}
+
+export interface UseTableHeaderGroupProps<D extends Record<string, unknown>> {
+  headers: Array<HeaderGroup<D>>;
+  getHeaderGroupProps: (
+    propGetter?: HeaderGroupPropGetter<D>,
+  ) => TableHeaderProps;
+  getFooterGroupProps: (
+    propGetter?: FooterGroupPropGetter<D>,
+  ) => TableFooterProps;
+  totalHeaderCount: number; // not documented
+}
+
+export interface UseTableColumnProps<D extends Record<string, unknown>> {
+  id: IdType<D>;
+  columns?: Array<ColumnInstance<D>> | undefined;
+  isVisible: boolean;
+  render: (type: 'Header' | 'Footer' | string, props?: object) => ReactNode;
+  totalLeft: number;
+  totalWidth: number;
+  getHeaderProps: (propGetter?: HeaderPropGetter<D>) => TableHeaderProps;
+  getFooterProps: (propGetter?: FooterPropGetter<D>) => TableFooterProps;
+  toggleHidden: (value?: boolean) => void;
+  parent?: ColumnInstance<D> | undefined; // not documented
+  getToggleHiddenProps: (userProps?: any) => any;
+  depth: number; // not documented
+  placeholderOf?: ColumnInstance | undefined;
+}
+
+export interface UseTableRowProps<D extends Record<string, unknown>> {
+  cells: Array<Cell<D>>;
+  allCells: Array<Cell<D>>;
+  values: Record<IdType<D>, CellValue>;
+  getRowProps: (propGetter?: RowPropGetter<D>) => TableRowProps;
+  index: number;
+  original: D;
+  id: string;
+  subRows: Array<Row<D>>;
+}
+
+export interface UseTableCellProps<D extends Record<string, unknown>, V = any> {
+  column: ColumnInstance<D>;
+  row: Row<D>;
+  value: CellValue<V>;
+  getCellProps: (propGetter?: CellPropGetter<D>) => TableCellProps;
+  render: (type: 'Cell' | string, userProps?: object) => ReactNode;
+}
+
+export type HeaderProps<D extends Record<string, unknown>> =
+  TableInstance<D> & {
+    column: ColumnInstance<D>;
+  };
+
+export type FooterProps<D extends Record<string, unknown>> =
+  TableInstance<D> & {
+    column: ColumnInstance<D>;
+  };
+
+export type CellProps<
+  D extends Record<string, unknown>,
+  V = any,
+> = TableInstance<D> & {
+  column: ColumnInstance<D>;
+  row: Row<D>;
+  cell: Cell<D, V>;
+  value: CellValue<V>;
+};
+
+export type Accessor<D extends Record<string, unknown>> = (
+  originalRow: D,
+  index: number,
+  sub: {
+    subRows: D[];
+    depth: number;
+    data: D[];
+  },
+) => CellValue;
+
+//#endregion
+
+// Plugins
+
+//#region useAbsoluteLayout
+export declare function useAbsoluteLayout<
+  D extends Record<string, unknown> = {},
+>(hooks: Hooks<D>): void;
+
+// export namespace useAbsoluteLayout {
+//   const pluginName = 'useAbsoluteLayout';
+// }
+
+// export declare const useAbsoluteLayout: (<
+//   D extends Record<string, unknown> = {},
+// >(
+//   hooks: Hooks<D>,
+// ) => void) & { pluginName: 'useAbsoluteLayout' };
+//#endregion
+
+//#region useBlockLayout
+export declare function useBlockLayout<D extends Record<string, unknown> = {}>(
+  hooks: Hooks<D>,
+): void;
+
+// export namespace useBlockLayout {
+//   const pluginName = 'useBlockLayout';
```

Comment:
these comments can be removed, no?

Reply:
Yes, they would be removed soon, along with any other testing code changes in this file.

Reply:
https://github.com/iTwin/iTwinUI/pull/1514/commits/280f183310fd777893f108e49121f5e8be4d97f2: Removed a few comments

I added a TODO (https://github.com/iTwin/iTwinUI/pull/1514#discussion_r1320051718) to reconfirm the file once again, which I am doing now.